homework.md

```diff
@@ -44,37 +44,37 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is the Programming Language for the Web. JavaScript can update and change both HTML and CSS
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+saving files on the cloud, working on a group , receiving feedback and reviews
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You copy it
 
 ## Question 10 
 
-What happens when you clone a repostory?
+what happens when you 'clone' a repository ?
 
-It send it to a friend
+It duplicate and download everything in the repository
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+When you get the last update on a branch 
 
```

Comment:
It is a method of submitting contributions to an open development project. It is called a pull request because you are asking the project to pull changes from your fork.